### DISCUSSION TITLE
Installing on ESXi 6.5 (Distributed Deployment)

### DISCUSSION THREAD

**Discussion by Jcremer11** [Security-Onion-Solutions/securityonion#2808] [Unsupported Versions] [answered]
Hello.
I see a few posts regarding SO v2 installation on ESXi. We are going to be installing SO v2 on ESXI 6.5, and I was wondering if anyone out there has had any experience with ESXi installs. Are there any settings you would recommend? Is it advisable to install VMWare Tools on the SO Master/Guest VM? I saw a post where someone recommended passing the PCI bus through to the sensor VM...but that's about it. Any recommendations would be greatly appreciated. 

thanks,
-Justin C.

**Comment by gcsrellis** [comment, marked as answer]
I've installed it on ESXi 5.5 and 6.7 and no issues.  I haven't installed tools and I'm not doing any hardware passthrough.  Works perfectly fine for me.  Just a standard installation like any other VM.

**Reply by Jcremer11** [reply]
Thanks for the reply. 

...and do you also install VMWare Tools on those VMs?

**Reply by gcsrellis** [reply]
Yes, I do install it.

**Reply by innovate-support** [reply]
My installs are exclusively on ESXi, different versions.  I have experienced repeated freezing of the VM's where Console access doesn't even work.  I've never figured out why though.  All the other VM's on the system never have any issues, only SO.  I've tried all sorts of different scenarios, some with more RAM, more CPU cores, plenty of Disk space, StandAlone and Distributed variants.  I don't do Full Packet Capture due to disk space consumption and I simply don't have the need for that in our environment right now.  I've installed open-vm-tools on them with no problems, but not installing it doesn't seem to make much different.  There may be some throughput advantage since you get different drivers, but I haven't investigated that either.  Short version is that it will work, but you may experience mysterious stability issues.
